wagtail/tests/dummy_sendfile_backend.py

```diff
@@ -0,0 +1,8 @@
+from django.http import HttpResponse
```

Comment:
This is just in case (convention) or there is a particular reason? I'm curious because for now, it just returns an ASCII string so it should not be needed, no?